NewSpring/ApollosProject
# [DEV-3820] Fixes SavePrayer update of Saved List

## DESCRIPTION

### What does this PR do, or why is it needed?

Currently when you save a prayer, it is not optimistically added to the savedPrayerList. This PR adds the needed `update` to the `savePrayer` mutation.

### How do I test this PR?

Go to a prayer menu, save a prayer, then make sure it's added to the savedPrayerList in that session.

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO


## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
